tracer/src/Datadog.Trace/RemoteConfigurationManagement/RemoteConfigurationManager.cs

```diff
@@ -187,6 +188,7 @@ private async Task Poll()
                 {
                     ProcessResponse(response, products);
                     _targetsVersion = response.Targets.Signed.Version;
+                    _backendClientState = response.Targets.Signed.Custom.OpaqueBackendState;
```

Comment:
Just wondering out loud, should we be more conservative with our expectations on the JSON, given the various issues we've seen. E.g. should we use null propagation?
```suggestion
                    _backendClientState = response.Targets.Signed.Custom?.OpaqueBackendState;
```

Reply:
Agreed. Fixed in cb30a93ff